### PR TITLE
#68 don't close RootCloseWrapper on right clicks

### PR DESCRIPTION
**Pull request by adamziel**
re #68

### CODE REVIEW

File: test/RootCloseSpec.js [hunk 1]
```diff
@@ -34,6 +34,19 @@ describe('RootCloseWrapper', function () {
...
+    let spy = sinon.spy();
+    render(
+      <RootCloseWrapper onRootClose={spy}>
+        <div id='my-div'>hello there</div>
+      </RootCloseWrapper>
+    , mountPoint);
+
+    simulant.fire(document.getElementById('my-div'), 'click', {button: 1});
+
+    expect(spy).to.have.been.called;
```
Comment (taion) [inline test/RootCloseSpec.js]: Should be the opposite, no?

Comment (adamziel) [inline test/RootCloseSpec.js]: Yup, thanks

File: src/RootCloseWrapper.js [hunk 2]
```diff
@@ -10,6 +10,14 @@ const CLICK_WAS_INSIDE = '__click_was_inside';
 
 let counter = 0;
 
+function isLeftClickEvent(event) {
+  return event.button === 0
```
Comment (taion) [inline src/RootCloseWrapper.js]: NIT: Do we use semicolons in this repo?

Comment (adamziel) [inline src/RootCloseWrapper.js]: Why so strict when semicolon insertion keeps you safe? :-)
http://www.ecma-international.org/ecma-262/5.1/#sec-7.9

Comment (taion) [inline src/RootCloseWrapper.js]: Yes, but we still want consistency in code style.

**Comment by jquense** [comment]
LGTM

**Comment by taion** [comment]
LGTM. @adamziel mind squashing?

**Comment by adamziel** [comment]
@taion squash :-)

**Comment by taion** [comment]
I'll squash and merge by hand then.

**Comment by taion** [comment]
Merged in https://github.com/react-bootstrap/react-overlays/commit/f1528af806236627df49e0f661aec7cb48980863.

**Comment by taion** [comment]
@jquense Can this be v0.6.1?

**Comment by jquense** [comment]
sure, no time to release now tho for me.

**Pull request closed without merging**